core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -565,6 +569,25 @@ oppia.directive('conversationSkin', [
                           newStateName, _nextFocusLabel) +
                         ExplorationPlayerService.getRandomSuffix());
                     }
+
+                    $http.get(EXPLORATION_SUMMARY_DATA_URL_TEMPLATE, {
+                      params: {
+                        stringified_exp_ids: JSON.stringify(
+                          [refresherExplorationId])
+                      }
+                    }).then(function(response) {
+                      if (response.data.summaries.length > 0) {
+                        RefresherExplorationConfirmationModalService.
+                          displayRedirectConfirmationModal(
+                            refresherExplorationId,
+                            function(
+                              redirectConfirmed) {
```

Comment:
Can move this to previous line.

Also, do you even need an arg to be passed? Couldn't you just have the callback trigger in the success case?

Reply:
Done, ya, I removed the arg